libraries/di/Dependencies.ts

```diff
@@ -27,3 +27,6 @@ export class Dependencies implements IDependencies {
     this.cache.clear();
   }
 }
+
+export { Dependencies };
```

Comment:
estamos utilizando este estilo para exportar

```suggestion
export class Dependencies {}
```

Reply:
cambios aplicados